public/app/plugins/datasource/grafana-azure-monitor-datasource/components/MetricsQueryEditor/MetricsQueryEditor.test.tsx

```diff
@@ -22,6 +22,7 @@ describe('Azure Monitor QueryEditor', () => {
         datasource={mockDatasource}
         variableOptionGroup={variableOptionGroup}
         onChange={() => {}}
+        setError={() => {}}
```

Comment:
Could we also have some tests here? Or do you plan to have them elsewhere?

Reply:
Yeah, tests were coming but wanted to get more feedback on my approach. seems good to continue with it for now